src/Native/Runtime/unix/UnwindHelpers.cpp

```diff
@@ -488,8 +728,9 @@ UnwindInfoSections LocateUnwindSections(uintptr_t pc)
 #else // __APPLE__
 
 #if _LIBUNWIND_SUPPORT_DWARF_UNWIND
-    dl_iterate_cb_data cb_data = {&uwInfoSections, pc };
-    dl_iterate_phdr(LocateSectionsCallback, &cb_data);
+    _addressSpace.findUnwindSections(pc, uwInfoSections);
+//    dl_iterate_cb_data cb_data = {&uwInfoSections, pc };
```

Comment:
Delete this and `LocateUnwindSections` since it is not needed anymore?

Reply:
It sounds like `StepFrame` can be changed to call findUnwindSections directly now, like:

```
if (!addressSpace.findUnwindSections(pc, uwInfoSections)
    return false;
```

and this method can be deleted too.

Reply:
Done!